assets/js/localization.js

```diff
@@ -207,17 +220,33 @@ function getLocales() {
         var localePairs = [];
         for(var code in locales) {
             localePairs.push([code, locales[code]]);
+            pageLocales[code] = locales[code];
```

Comment:
It sounds like this data could also be stored inside the locale dropdown's items below at L242. I'm also not sure why this is called `pageLocales`.